### PR TITLE
Fix PCR-HH Rate Calculation

### PR DESCRIPTION
**Pull request by abelzer212**
## Purpose

Make the outlier rate consistent between PCR-AD and PCR-HH

#### Linked Issues to Close

https://qmacbis.atlassian.net/browse/OY2-18570

#### Pull Request Creator Checklist

- [x] This PR has an associated issue or issues.
- [x] The associated issue(s) are linked above.
- [x] This PR meets all acceptance criteria for those issues.
- [x] This PR and linked issue(s) are adequately documented
- [x] This PR and linked issues(s) are a complete description of the changeset; an individual or team should be able to understand the issue(s) and changes by reading through this PR and it's links, with no further interaction.
- [x] Someone has been assigned this PR.
- [x] At least one person has been marked as reviewer on this PR.

#### Pull Request Reviewer/Assignee Checklist

- [ ] This PR has an associated issue or issues.
- [ ] The associated issue(s) are linked above.
- [ ] This PR meets all acceptance criteria for those issues.
- [ ] This PR and linked issue(s) are adequately documented
- [ ] This PR and linked issues(s) are a complete description of the changeset; an individual or team should be able to understand the issue(s) and changes by reading through this PR and it's links, with no further interaction.

### CODE REVIEW

File: services/ui-src/src/components/PCRRate/index.tsx [hunk 1]
```diff
@@ -68,7 +64,7 @@ export const PCRRate = ({
       denominator: 6,
       rateIndex: 8,
       multiplier: 1000,
-      rateDecimals: measureId === "PCR-HH" ? 4 : 1,
+      rateDecimals: 1,
```
Comment (jphechter) [inline services/ui-src/src/components/PCRRate/index.tsx:67]: This change will cause test failures in both `PCRHH.spec.ts` and `PCRRate/index.test.ts`. Should be as simple as dropping a couple decimal places.

Not sure what line in the Cypress test, but the PCR-HH specific section of the unit test is line `154` onward.

**Review by jphechter — CHANGES_REQUESTED** [review]
tests need to be updated and then this should be good

File: services/ui-src/src/components/PCRRate/index.tsx (hunk 1 above)
Comment (abelzer212) [reply services/ui-src/src/components/PCRRate/index.tsx:67]: Ahh good to know I'll find that in the code

**Review by jdinh8124 — DISMISSED** [review]

**Review by jphechter — APPROVED** [review]
LGTM! 👍

**Pull request merged**